spec/aged_brie_spec.js

```diff
@@ -1,52 +1,57 @@
 describe("Aged Brie", function() {
+    const anonymousName = "Aged Brie";
+    const anonymousSellIn = 5;
+
     beforeEach(function() {
-        brie1 = new AgedBrie("Aged Brie", 4, 2);
-        brie2 = new AgedBrie("Aged Brie", 4, 50);
-        gildedRose = new Inn([brie1, brie2]);
+        BRIE_WITH_SELLIN_4_DAYS = new AgedBrie("sellIn_4", 4, 2);
```

Comment:
Nice :+1: Would it be any more or less clear to call this something like `BRIE_EXPIRING_IN_4_DAYS`?